Gemfile.lock

```diff
@@ -845,7 +845,7 @@ DEPENDENCIES
   web-console (= 4.0.4)
 
 RUBY VERSION
-   ruby 2.7.5p203
+   ruby 3.0.2p107
```

Comment:
This file shouldn't''t be in the PR, we don't want to change the ruby version here